### PR TITLE
make module to display happy stories

### PR DESCRIPTION
**Pull request by AlexeyKasaev3**
HTML разметка для отображения списка счастливых историй

### CODE REVIEW

File: right-for-life_front/src/app/happy-stories-list/happyStoriesList.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  const [ happyStoriesData, setHappyStoriesData ] = useState(null);
+  const [ isStoriesDisplayed, setIsStoriesDisplayed ] = useState(false);
+
+  function handleShowStoriesButtonClick() {
+    setIsStoriesDisplayed(!isStoriesDisplayed);
+  }
+
+  useEffect(() => {
+    fetch('https://raw.githubusercontent.com/AlexeyKasaev3/softServe-academy/master/demo-3-data/news.json')
```
Comment (PsyhoBelka) [inline right-for-life_front/src/app/happy-stories-list/happyStoriesList.js]: i think we must wait until all data-JSONs will be added from others and then use them

File: right-for-life_front/src/app/happy-stories-list/happyStoriesList.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { HappyStoriesListItem } from '../happy-stories-list-item';
```
Comment (PsyhoBelka) [inline right-for-life_front/src/app/happy-stories-list/happyStoriesList.js]: do not named folder with -, use CamelCase instead

Comment (AlexeyKasaev3) [reply right-for-life_front/src/app/happy-stories-list/happyStoriesList.js]: Got it

Comment (AlexeyKasaev3) [reply right-for-life_front/src/app/happy-stories-list/happyStoriesList.js]: Should I fix it now?

Comment (PsyhoBelka) [reply right-for-life_front/src/app/happy-stories-list/happyStoriesList.js]: yeap! fix-commit-push, it's natural lifecycle:)

File: right-for-life_front/src/components/NewsList/ArticlesList.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (protonaby) [inline right-for-life_front/src/components/NewsList/ArticlesList.jsx]: Please check folder naming.
Expected: NewsList -> ArticlesList
Actual: AnimalsList -> ArticlesList

Comment (AlexeyKasaev3) [reply right-for-life_front/src/components/NewsList/ArticlesList.jsx]: Done, thanks

File: right-for-life_front/src/containers/AnimalsPage/AnimalsPage.jsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
-import { AnimalsList } from "../../components/AnimalsList";
+import { AnimalsList } from "../../components/ArticleList";
```
Comment (protonaby) [inline right-for-life_front/src/containers/AnimalsPage/AnimalsPage.jsx]: Please make sure your code compiles before submitting for review

File: right-for-life_front/src/containers/HappyStoriesPage/HappyStoriesPage.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React, {useState, useEffect} from "react";
+import { ArticlesList } from "../../components/NewsList";
```
Comment (protonaby) [inline right-for-life_front/src/containers/HappyStoriesPage/HappyStoriesPage.jsx]: NewsList -> ArticlesList

File: right-for-life_front/src/containers/NewsPage/NewsPage.jsx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, {useState, useEffect} from "react";
-import {NewsList} from "../../components/NewsList";
+import {ArticlesList} from "../../components/NewsList";
```
Comment (protonaby) [inline right-for-life_front/src/containers/NewsPage/NewsPage.jsx]: NewsList -> ArticlesList

File: right-for-life_front/src/containers/HappyStoriesPage/HappyStoriesPage.jsx (hunk 5 above)
Comment (AlexeyKasaev3) [reply right-for-life_front/src/containers/HappyStoriesPage/HappyStoriesPage.jsx]: Done

File: right-for-life_front/src/containers/NewsPage/NewsPage.jsx (hunk 6 above)
Comment (AlexeyKasaev3) [reply right-for-life_front/src/containers/NewsPage/NewsPage.jsx]: Done

**Pull request merged**